### PR TITLE
All my pull requests bundled into one

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**
I merged all my branches with open PRs together and resolved all the conflicts. This branch is the result. If you guys approve all my other PRs that are open as of this writing, we can just merge this one and close the rest. If any of them need changes, I can merge the changes into this branch.

### CODE REVIEW

**Review by trevorhuis-smoothstack — APPROVED** [review]
It looks like every pull request has an approval!

**Comment by justin-obrien-smoothstack** [comment]
I feel like PRs should be approved by both reviewers before being merged. If @sean-darsie approves this one I'll merge it (unless one of you merges it first).

File: src/main/java/com/ss/training/lms/controller/AdminBranchController.java
Comment (sean-darsie) [inline src/main/java/com/ss/training/lms/controller/AdminBranchController.java:106]: should this endpoint also have /update on it?

File: src/main/java/com/ss/training/lms/controller/AdminBranchController.java
Comment (sean-darsie) [inline src/main/java/com/ss/training/lms/controller/AdminBranchController.java:71]: This should not have the same path as the delete path. Could add delete after the id to distinguish them

File: src/main/java/com/ss/training/lms/controller/AdminAuthorController.java
Comment (sean-darsie) [inline src/main/java/com/ss/training/lms/controller/AdminAuthorController.java:106]: The path should indicate that you are doing an update

**Review by sean-darsie — APPROVED** [review]
I've reviewed all of these changes individually. Should be good.

**Comment by justin-obrien-smoothstack** [comment]
I think the paths are fine because the request method will determine what operation is performed.

**Pull request merged**
